Readme.md

```diff
@@ -0,0 +1,905 @@
+
```

Comment:
This file, *assuming* that it is a Spanish translation of README.md needs some fixes:

- Rename the file to something more appropriate, like `README.es.md`
- Add the badges and other various formatting back